extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/MapReduceOptions.java

```diff
@@ -25,8 +25,6 @@ public class MapReduceOptions {
     private long maxTime;
     private MapReduceAction action;
     private String databaseName;
-    private boolean sharded;
-    private boolean nonAtomic;
```

Comment:
There are no replacement or additional fields we need to support?

Reply:
According to the 4.11.1 Javadocs these two options aren't supported on a database level from MongoDB 4.4 onwards.

In comparison to 4.11.1 only those two fields were removed and none were added.

Reply:
I believe there will be more work that needs to be invested in this area in the future.
The whole Map-Reduce functionality was deprecated in version 5.0 and should be replaced with aggregates.
See [Map-Reduce](https://www.mongodb.com/docs/v5.1/core/map-reduce/#:~:text=Starting%20in%20MongoDB%205.0%2C%20map,%2C%20%24merge%20%2C%20and%20others.).